### ISSUE TITLE
Why are you loading FLUX.1-dev instead of FLUX.1-Fill-dev?

### ISSUE THREAD

**Issue by jroessler** [nftblackmagic/catvton-flux#55]
In your inference script (tryon_inference.py), why are you loading "FLUX.1-dev" instead of "FLUX.1-Fill-dev"? Doesn't it make sense to load FLUX.1-Fill-dev? 

```
pipe = FluxFillPipeline.from_pretrained(
    "black-forest-labs/FLUX.1-dev",
    transformer=transformer,
    torch_dtype=torch.bfloat16
).to("cuda")
```

Same goes for training script (train_flux_inpaint.py), where you set MODEL_NAME="black-forest-labs/FLUX.1-dev".

Thanks!

**Comment by jroessler** [comment]
Okay I partially got it! In your training script, you're using all the components (text encoders, vae, schedulers) from FLUX.1-dev and only the transformer from xiaozaa/flux1-fill-dev-diffusers, which is based on FLUX.1-Fill-dev, I guess.

But why are you not using FLUX.1-Fill, like so:

`pipe = FluxFillPipeline.from_pretrained("black-forest-labs/FLUX.1-Fill-dev", torch_dtype=torch.bfloat16).to("cuda")`

Is there any advantage / important consideration in using your approach?
